### PR TITLE
Blog App: Views

### PR DESCRIPTION
**Pull request by albertkantwi**
Within this pull request, I have incorporated the following modifications;

- Added root route.
- Implement basic UI for users.
- Implements basic UI for posts.

Kindly review for me.

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,2 +1,32 @@
...
+</div>
+
+<div class="user-posts">
+  <% @user.posts.limit(3).each do |post| %>
+    <div class="post">
+      <h2><%= link_to "#{post.title}", user_post_path(post.author, post) %></h2>
+      <p><%= post.text %></p>
+      <p class="counter">Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %></p>
+    </div>
+  <% end %>
```
Comment (BenMukebo) [inline app/views/users/show.html.erb]: * [ ]  Please make sure to use the `most_recent_posts` method you created in the last milstone in your User model to render the 3 most recent posts as mentioned in the project requirements. Do this in your user show page.
```ruby
# app/views/users/show.html.erb

[...]

<% @user.recent_post.each do |post| %>
```

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @albertkantwi,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻
- No linter errors. ✔️ 
- Gitflow was correctly used.  ✔️   
- Good commit messages ✔️ 
- **See all Post** links opens user's post index as displayed on the wireframe. 👍🏻 
- Uses `post.recent_comments` render 5 most recent comments. 👍🏻 
- Post show page displays post with all comments listed as shown on the wireframe. 🥇 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/show.html.erb (hunk 1 above)
Comment (albertkantwi) [reply app/views/users/show.html.erb]: Hello @BenMukebo I just added the changes and it's working. 
Kindly review for me.

**Review by Deepakdanger — APPROVED** [review]
Hi @albertkantwi, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**